doc/usecases/snippet_sharing/UC_edit_snippet.md

```diff
@@ -0,0 +1,19 @@
+# Use Case: Edit configuration snippet
+
+## Summary
+
+Title: Edit configuration snippet
+Scope: Entry Management
+Level: User Goal
+Actors: Authenticated user (snippet owner), Moderator
+Brief: Users can edit their own database entries. Moderators can edit all configuration snippets in the database, also the ones of other users.

```

Comment:
> Help system in what sense?

I mean beyond small hover texts but descriptions of configuration format syntax, limitations and so on. Plugins should document this as part of their README.md so maybe it is easiest to simply show the rendered README.md (if someone wants to understand what a configuration file format is capable of doing).